### ISSUE TITLE
JSON exception with some songs and occasional crashes

### ISSUE THREAD

**Issue by chusopr** [chusopr/plugin.video.lastfm#2]
```json
{
  "playlist": [
    {
      "_blacklisted": null,
      "_name": "Háblame de ti",
      "url": "/music/Los+Pecos/_/H%C3%A1blame+de+ti",
      "_wiki": null,
      "duration": 213,
      "stats": null,
      "user_stats": null,
      "listener_stats": null,
      "similarity_score": null,
      "artists": [
        {
          "_blacklisted": null,
          "_name": "Los Pecos",
          "url": "/music/Los+Pecos",
          "_wiki": null,
          "factboxes": null,
          "similarity_score": null,
          "main_image": null,
          "header_image": null,
          "playlinks": null,
          "stats": null,
          "tags": null,
          "user_stats": null,
          "on_tour": null,
          "event_years": null,
          "user_tags": null,
          "listener_trend": null,
          "gallery": null,
          "correction_suggestions": null,
          "shoutbox_permissions": null,
          "metalinks": null,
          "mainstream_percentage": null,
          "popularity_rank": null,
          "user_last_scrobble": null,
          "chart_position": null,
          "is_bookmarked": null,
          "similar_artists": null,
          "canonical": null,
          "top_tracks": null,
          "popular_track": null,
          "top_albums": null,
          "popular_albums": null,
          "latest_release": null,
          "top_listeners": null,
          "listening_now": null,
          "listeners_you_know": null,
          "events": null,
          "nearby_events": null,
          "_shouts": null,
          "wiki_history": null,
          "wiki": null,
          "name": "Los Pecos"
        }
      ],
      "_playlinks": [
        {
          "affiliate": "youtube",
          "id": "awl9GIbbArA",
          "url": "https://www.youtube.com/watch?v=awl9GIbbArA",
          "source": "ingested"
        }
      ],
      "user_events": null,
      "tags": null,
      "_buylinks": null,
      "metalinks": null,
      "user_tags": null,
      "listener_trend": null,
      "correction_suggestions": null,
      "shoutbox_permissions": null,
      "lyrics": null,
      "partial_release_date": null,
      "label": null,
      "chart_position": null,
      "is_bookmarked": null,
      "similar_tracks": null,
      "canonical": null,
      "primary_album": null,
      "top_albums": null,
      "listeners_you_know": null,
      "_shouts": null,
      "wiki_history": null,
      "wiki": null,
      "name": "Háblame de ti",
      "playlinks": [
        {
          "affiliate": "youtube",
          "id": "awl9GIbbArA",
          "url": "https://www.youtube.com/watch?v=awl9GIbbArA",
          "source": "ingested"
        }
      ]
    }
  ]
}
```
Causes:
```
ERROR: query " Los Pecos - Háblame de ti": Failed to parse JSON  (caused by ValueError('No JSON object could be decoded',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by chusopr** [comment]
This is an issue with youtube-dl upstream, can be easily reproduced with their CLI:
```
youtube-dl 'ytsearch:The Jolly Boys Rehab'
```

**Comment by chusopr** [comment]
This seems to be caused by the leading space after `ytsearch:` in https://github.com/chusopr/plugin.video.lastfm/blob/60f428559e854165f654efa735d65f0ace58858d/addon.py#L214
```
$ youtube-dl 'ytsearch: Alan Walker - The Spectre'
[youtube:search] query " Alan Walker - The Spectre": Downloading page 1
ERROR: query " Alan Walker - The Spectre": Failed to parse JSON  (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)')); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
$ youtube-dl 'ytsearch:Alan Walker - The Spectre'
[youtube:search] query "Alan Walker - The Spectre": Downloading page 1
[download] Downloading playlist: Alan Walker - The Spectre
[youtube:search] playlist Alan Walker - The Spectre: Collected 1 video ids (downloading 1 of them)
[download] Downloading video 1 of 1
[youtube] wJnBTPUQS5A: Downloading webpage
```

**Comment by chusopr** [comment]
Fixed upstream (3de8e6a).
